srml/staking/src/tests.rs

```diff
@@ -523,28 +523,34 @@ fn nominating_and_rewards_should_work() {
+		let payout_for_20 = 2 * total_payout_1 / 3;
 		if cfg!(feature = "equalize") {
-			// Nominator 2: has [400/2000 ~ 1/5 from 10] + [600/2000 ~ 3/10 from 20]'s reward.
-			assert_eq!(Balances::total_balance(&2), initial_balance + payout_for_10/5 + payout_for_20*3/10 - 1);
-			// Nominator 4: has [400/2000 ~ 1/5 from 20] + [600/2000 ~ 3/10 from 10]'s reward.
-			assert_eq!(Balances::total_balance(&4), initial_balance + payout_for_20/5 + payout_for_10*3/10);
+			// Nominator 2: has [400 / 2000 ~ 1 / 5 from 10] + [600 / 2000 ~ 3 / 10 from 20]'s reward.
+			assert_eq!(Balances::total_balance(&2), initial_balance + payout_for_10 / 5 + payout_for_20 * 3 / 10);
+			// Nominator 4: has [400 / 2000 ~ 1 / 5 from 20] + [600 / 2000 ~ 3 / 10 from 10]'s reward.
+			assert_eq!(Balances::total_balance(&4), initial_balance + payout_for_20 / 5 + payout_for_10 * 3 / 10);
 
 			// Validator 10: got 1000 / 2000 external stake.
-			assert_eq!(Balances::total_balance(&10), initial_balance + payout_for_10/2);
+			assert_eq!(Balances::total_balance(&10), initial_balance + payout_for_10 / 2);
 			// Validator 20: got 1000 / 2000 external stake.
-			assert_eq!(Balances::total_balance(&20), initial_balance + payout_for_20/2);
+			assert_eq!(Balances::total_balance(&20), initial_balance + payout_for_20 / 2);
 		} else {
-			// Nominator 2: has [400/1800 ~ 2/9 from 10] + [600/2200 ~ 3/11 from 20]'s reward. ==> 2/9 + 3/11
-			assert_eq!(Balances::total_balance(&2), initial_balance + (2*payout_for_10/9 + 3*payout_for_20/11) - 2);
-			// Nominator 4: has [400/1800 ~ 2/9 from 10] + [600/2200 ~ 3/11 from 20]'s reward. ==> 2/9 + 3/11
-			assert_eq!(Balances::total_balance(&4), initial_balance + (2*payout_for_10/9 + 3*payout_for_20/11) - 2);
-
-			// Validator 10: got 800 / 1800 external stake => 8/18 =? 4/9 => Validator's share = 5/9
-			assert_eq!(Balances::total_balance(&10), initial_balance + 5*payout_for_10/9 - 1);
-			// Validator 20: got 1200 / 2200 external stake => 12/22 =? 6/11 => Validator's share = 5/11
-			assert_eq!(Balances::total_balance(&20), initial_balance + 5*payout_for_20/11);
+			// Nominator 2: has [400 / 1800 ~ 2 / 9 from 10] + [600 / 2200 ~ 3 / 11 from 20]'s reward. ==> 2 / 9 + 3 / 11
+			assert_eq!(
+				Balances::total_balance(&2),
+				initial_balance + (2 * payout_for_10 / 9 + 3 * payout_for_20 / 11) - 2
+			);
+			// Nominator 4: has [400 / 1800 ~ 2 / 9 from 10] + [600 / 2200 ~ 3 / 11 from 20]'s reward. ==> 2 / 9 + 3 / 11
+			assert_eq!(
+				Balances::total_balance(&4),
+				initial_balance + (2 * payout_for_10 / 9 + 3 * payout_for_20 / 11) - 2
+			);
+
+			// Validator 10: got 800 / 1800 external stake => 8 / 18 =? 4 / 9 => Validator's share = 5 / 9
+			assert_eq!(Balances::total_balance(&10), initial_balance + 5*payout_for_10 / 9 - 1);
+			// Validator 20: got 1200 / 2200 external stake => 12 / 22 =? 6 / 11 => Validator's share = 5 / 11
+			assert_eq!(Balances::total_balance(&20), initial_balance + 5*payout_for_20 / 11);
```

Comment:
```suggestion
			assert_eq!(Balances::total_balance(&20), initial_balance + 5*payout_for_20 / 11 + 1);
```
this is required to make the test `cargo test --release --verbose --no-default-features --features std` works, I don't know why it has changed though